### PR TITLE
add update infos, and consommation infos too

### PR DESCRIPTION
**Pull request by mathildegrimal**
#### Before submitting, make sure:
- [x] You have rebased your branch
- [x] You describe your changes
- [x] You describe How to test
- [ ] I work on the same file as one of my colleague which one?

#### Describe below your changes:
Capture d’écran 2022-02-27 à 14 51 58
Capture d’écran 2022-02-27 à 14 52 09
Capture d’écran 2022-02-27 à 14 51 45

##### Describe below How to test:
⚠Before testing you have to pull get-user-stats branch from back-end

Then go to /moncompte page and connect
you ll have access to stats data, user infos and update 

This is experimental because I couldn't figure out how to save new informations from update into the user's state

### CODE REVIEW

File: src/users/components/accountDashboard/AccountDashboard.tsx [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  switch (component) {
+    case 'conso':
+      componentToDisplay = <Consommation id={user.id?user.id : ""} />;
+      break;
+    case 'infos':
+      componentToDisplay = <Informations user={user} />;
+      break;
+    case 'update':
+      componentToDisplay = <UpdateInformations user={user} />;
+      break;
```
Comment (Antonin-dev) [inline src/users/components/accountDashboard/AccountDashboard.tsx:27]: i like that

Comment (mathildegrimal) [reply src/users/components/accountDashboard/AccountDashboard.tsx:23]: cool ! j'avais peur que tu rales la dessus je savais pas comment faire autrement

**Comment by mathildegrimal** [comment]
[En tant qu'utilisateur je souhaite pouvoir modifier mes informations de compte (nom, prénom, mot de passe)](https://trello.com/c/gkCAaQed/96-en-tant-quutilisateur-je-souhaite-pouvoir-modifier-mes-informations-de-compte-nom-pr%C3%A9nom-mot-de-passe)

File: src/users/store/userStore.ts [hunk 2]
```diff
@@ -19,12 +19,23 @@ export const userSlice = createSlice({
...
+      } else {
+        state.value = { ...action.payload };
+        state.error = null;
+      }
+    },
     setUserError: (state) => {
       state.error = 'Email deja existant';
     },
+    setPasswordUpdateError: (state) => {
+      state.error = "L'ancien mot de passe est requis";
```
Comment (mathildegrimal) [inline src/users/store/userStore.ts:34]: I have to delete that I don't use it

File: src/UI/ButtonCustom.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import Button from '@mui/material/Button';
+
+const ButtonCustom = (props: any) => {
```
Comment (Antonin-dev) [inline src/UI/ButtonCustom.tsx:3]: {type, disable onclick}
maybe you ban déstructure this

File: src/users/components/accountDashboard/AccountDashboard.tsx (hunk 1 above)
Comment (Antonin-dev) [reply src/users/components/accountDashboard/AccountDashboard.tsx:23]: yep c'est top

**Pull request merged**